### PR TITLE
Add note about np.sum

### PR DESCRIPTION
**Pull request by guilhermeleobas**
Following up on #4386. This PR adds a note about `np.sum` not supporting tuples on its `axis` parameter.

### CODE REVIEW

**Comment by guilhermeleobas** [comment]
I don't think my changes are breaking the CI.

File: docs/source/reference/numpysupported.rst [hunk 1]
```diff
@@ -210,7 +210,7 @@ The following methods of Numpy arrays are supported:
 * :meth:`~numpy.ndarray.repeat` (no axis argument)
 * :meth:`~numpy.ndarray.reshape` (only the 1-argument form)
 * :meth:`~numpy.ndarray.sort` (without arguments)
-* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` argument)
+* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` argument. ``axis`` only supports scalar values)
```
Comment (stuartarchibald) [inline docs/source/reference/numpysupported.rst]: I think s/scalar/``integer`` to be really explicit. Also, this line needs wrapping.

File: docs/source/reference/numpysupported.rst [hunk 2]
```diff
@@ -210,7 +210,7 @@ The following methods of Numpy arrays are supported:
 * :meth:`~numpy.ndarray.repeat` (no axis argument)
 * :meth:`~numpy.ndarray.reshape` (only the 1-argument form)
 * :meth:`~numpy.ndarray.sort` (without arguments)
-* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` argument)
+* :meth:`~numpy.ndarray.sum` (with or without the ``axis`` argument. ``axis`` only supports ``integer`` values)
```
Comment (stuartarchibald) [inline docs/source/reference/numpysupported.rst]: As noted here https://github.com/numba/numba/pull/4397#discussion_r310565569, this line is now too long ( >80 chars), please could you wrap it, thanks.

**Comment by stuartarchibald** [comment]
Thanks for the patch.

**Pull request merged**